crypto/mod.ts

```diff
@@ -158,16 +166,20 @@ const webCryptoDigestAlgorithms = [
 ] as const;
 
 type DigestAlgorithmName = WasmDigestAlgorithm;
+type FNVAlgorithms = "FNV32" | "FNV32A" | "FNV64" | "FNV64A";
 
 type DigestAlgorithmObject = {
   name: DigestAlgorithmName;
   length?: number;
 };
 
-type DigestAlgorithm = DigestAlgorithmName | DigestAlgorithmObject;
+type DigestAlgorithm =
+  | DigestAlgorithmName
+  | DigestAlgorithmObject
+  | FNVAlgorithms;
```

Comment:
I think `FNVAlgotithms` should be appended to `DigestAlgorithmName` type instead of `DigestAlgorithm` like:

```ts
type DigestAlgorithmName = WasmDigestAlgorithm | FNVAlgorithms;
```

With that change, we can call `digest` like `crypto.subtle.digest({ name: "FNV32" }, data)` (Current implementation only allows `crypto.subtle.digest("FNV32", data)` signature)

Reply:
Right, when I tried that Typescript complained about a lot of types at the test file. But I'll try it again anyway.

Reply:
again, why can't `FNVAlgorithms` be added to `DigestAlgorithmName`?

Reply:
Oh, I misunderstood the previous comment, sorry. I thought I made the right change but I didn't :x